src/app/tests/suites/certification/Test_TC_MWOCTRL_2_2.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 263.1.1. [TC-MWOCTRL-2.2] Secondary functionality with DUT as Server
+
+PICS:
+    - MWOCTRL.S
+    - MWOCTRL.S.A0003
+
+config:
+    nodeId: 0x12344321
+    cluster: "Microwave Oven Control"
+    endpoint: 1
+
+tests:
+    - label: "Step 1: Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Step 2: Read the CookTime attribute"
+      command: "readAttribute"
+      attribute: "CookTime"
+      response:
+          saveAs: CookTimeValue
+          constraints:
+              type: elapsed-s
+              minValue: 1
+              maxValue: 86400
+
+    - label: "Step 3: Read the MinPower attribute"
+      command: "readAttribute"
+      attribute: "MinPower"
+      response:
+          saveAs: MinPowerValue
+          constraints:
+              type: int8u
+              minValue: 1
+              maxValue: 100
+
+    - label: "Step 4: Read the MaxPower attribute"
+      command: "readAttribute"
+      attribute: "MaxPower"
+      response:
+          saveAs: MaxPowerValue
+          constraints:
+              type: int8u
+              minValue: MinPowerValue
+              maxValue: 100
+
+    - label: "Step 5: Read the PowerStep attribute"
+      command: "readAttribute"
+      attribute: "PowerStep"
+      response:
+          saveAs: PowerStepValue
+          constraints:
+              type: int8u
+              minValue: MinPowerValue
+              maxValue: MaxPowerValue
+
+    - label: "Step 6: Read the PowerSetting attribute"
+      command: "readAttribute"
+      attribute: "PowerSetting"
+      response:
+          saveAs: PowerSettingValue
+          constraints:
+              type: int8u
+              minValue: MinPowerValue
+              maxValue: MaxPowerValue
+
+    - label: "Step 7: Send the SetCookingParameters command"
+      command: "SetCookingParameters"
+      arguments:
+          values:
+              - name: PowerSetting
+                value: MinPowerValue
```

Comment:
The value for PowerSetting should not be MinPowerValue as per the test plan - " TH sendsSetCookingParametersto DUT with CookTime set to 60 and PowerSetting set to PowerValue%MaxPowervalue + PowerStepValue"